lib/views/demo_screens/explore_events_demo.dart

```diff
@@ -71,6 +71,10 @@ class DemoExploreEvents extends StatelessWidget {
       // else render refresh icon along with the list of searched events for exploration.
       body: Stack(
         children: [
+          SizedBox(
+            width: SizeConfig.screenWidth,
+            height: SizeConfig.screenHeight,
+          ),
```

Comment:
Remove this and use "Expanded"

Reply:
can i use flex as Expanded is giving errors